xonsh/tools.py

```diff
@@ -555,6 +555,21 @@ def ensure_int_or_slice(x):
         return int(x)
 
 
+def is_completions_display_value(x):
+    return x in {'none', 'single', 'multi'}
+
+
+def to_completions_display_value(x):
+    if str(x).lower() in {'none', 'false'}:
+        return 'none'
+    if str(x).lower() in {'multi', 'true'}:
+        return 'multi'
+    if str(x).lower() in {'single'}:
+        return 'single'
+    print('Warning: \'{}\' is not a valid value for $COMPLETIONS_DISPLAY. Using \'multi\'.'.format(str(x)))
```

Comment:
Please use a the Python standard library `warn()` function here.  Also you can wrap in double quotes so that you don't have to escape the internal single quotes.  Also this line is too long and you don't need to call `str()` on x, format does that for you.